src/runtime-rs/crates/agent/src/lib.rs

```diff
@@ -38,6 +38,7 @@ pub trait AgentManager: Send + Sync {
     async fn start(&self, address: &str) -> Result<()>;
     async fn stop(&self);
 
+    // agent_sock returns the socket address `with default server port`
```

Comment:
Delete the back slash char around `with default server port`

Reply:
Fixed